src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.Executor.cs

```diff
@@ -51,6 +58,52 @@ internal partial class DiagnosticIncrementalAnalyzer
             {
                 throw ExceptionUtilities.Unreachable;
             }
+
+            static bool IsAnalyzerSuppressedForDocument(
```

Comment:
can we switch this to be IsAnalzyerEnabled, and then if someone wants to bail out they do `if (!IsAnalzyerEnabled`?  I find the negatives really hard to track.

Reply:
Sure, done.